### DISCUSSION TITLE
Access Point "OpenDTU-*" default password "openDTU42" funktioniert nicht

### DISCUSSION THREAD

**Discussion by SCAfeMM** [tbnobody/OpenDTU#669] [Q&A] [answered]
Hallo zusammen,

habe für die DTU alles zusammen gebaut und die "webapp: add app.js.gz #538" installiert.

Ich bekomme den Access Point called "OpenDTU-*" angezeigt, jedoch funktioniert das default password "openDTU42" nicht.
Hat jemand eine Idee?

viele Grüße
Andreas

**Comment by Solarteur** [comment, marked as answer]
Hast Du diesen Eintrag
https://github.com/tbnobody/OpenDTU/issues/226
gelesen? Speziell die Einträge ab 9. Nov. 2022.

**Reply by SCAfeMM** [reply]
das hatte ich noch nicht gelesen, danke für die Info!
